client/asset/btc/btc.go

```diff
@@ -350,6 +350,8 @@ type BTCCloneCFG struct {
 	TxVersion func() int32
 	// ManualMedianTime causes the median time to be calculated manually.
 	ManualMedianTime bool
+	// OmitRPCOptionsArg is for clones that don't take an otpions argument.
+	OmitRPCOptionsArg bool
```

Comment:
I can live with this for now, but it's almost certainly too broad now that I consider it more.  Some RPCs may have options but not other RPCs.  We're just intending this for use with the one fundrawtransaction RPC.

Also, there's a typo here "otpions"